.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
```

Comment:
add node_modules to .gitignore

Reply:
It is there. I had some inconsistency with end-of-line characters between lint and git settings, that's why git thought I removed it and then added it back, while in fact I didn't touch it.